client/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "plugins": ["emotion"]
```

Comment:
What is this change? :)

Reply:
I followed a tutorial in styling checkboxes. It was at first not clear that the tutorial used older bibs similar to @emotion/styled. I found another simplified way to style my component- will remove this. thank you.